docs/reference/ccr/overview.asciidoc

```diff
@@ -29,6 +29,41 @@ Replication can be configured in two ways:
 
 NOTE: You must also <<ccr-requirements,configure the leader index>>.
 
+When you initiate replication either manually or through an auto-follow pattern, the
+follower index is created on the local cluster. Once the follower index is created,
+the <<remote-recovery, remote recovery>> process copies all of the Lucene segment
+files from the remote cluster to the local cluster.
+
+When initiating following manually using the API, by default the recovery process
+will be asynchronous in relationship to the
+{ref}/ccr-put-follow.html[create follower request]. The request will return prior
+to the <<remote-recovery, remote recovery>> process being completed. If you
+would like to wait on the process to complete, you can use the
+`wait_for_active_shards` parameter to do so.
+
+//////////////////////////
+
+[source,js]
+--------------------------------------------------
+PUT /follower_index/_ccr/follow?wait_for_active_shards=1
+{
+  "remote_cluster" : "remote_cluster",
+  "leader_index" : "leader_index"
+}
+--------------------------------------------------
+// CONSOLE
+// TESTSETUP
+// TEST[setup:remote_cluster_and_leader_index]
+
+[source,js]
+--------------------------------------------------
+POST /follower_index/_ccr/pause_follow
+--------------------------------------------------
+// CONSOLE
+// TEARDOWN
+
+//////////////////////////
+
 [float]
 === The mechanics of replication
```

Comment:
I would suggest putting this content into a separate page (akin to the "How it works" pages in the other sections), since it's quite low level for an overview. That can be done in a separate PR if necessary.

Reply:
I have added a task to the meta issue #35975